scripts/js/lib/links/InternalLink.ts

```diff
@@ -61,7 +61,11 @@ export class InternalLink {
     if (this.value === "") {
       return [originFile];
     }
-    if (this.value.startsWith("/images") || this.value.startsWith("/videos")) {
+    if (
+      this.value.startsWith("/images") ||
+      this.value.startsWith("/videos") ||
+      (this.value.startsWith("/open-source") && this.value.endsWith(".pdf"))
```

Comment:
@arnaucasau this is more future proof if we upload other files in the future. We'll always store them in `public/`. Lmk what you think.

```suggestion
      this.value.endsWith(".pdf")
```